tasks/update/task.yaml

```diff
@@ -2,37 +2,41 @@ summary: Update Linode image.
 
 execute: |
 
-    case $SPREAD_REBOOT in
+    if [[ "$SPREAD_SYSTEM" = fedora-* ]]; then
```

Comment:
Can you please use a switch/case statement here. In all the cases where we special case Fedora or Ubuntu the `else` part is always broken when we add something new.